source/tachyons/_custom.css

```diff
@@ -37,3 +42,11 @@ a {
 .transition-300 {
   transition: 0.3s ease
 }
+
+h1, h2, h3, h4, h5, h6 {
```

Comment:
Not sure about these bottom two. They make everything look like the old site but I feel a little uneasy. What do you think? @andrewMacmurray